shell/components/form/PodAffinity.vue

```diff
@@ -125,6 +133,14 @@ export default {
     hasNamespaces() {
       return this.allNamespaces.length;
     },
+
+    namespaceModeLabel() {
```

Comment:
Hi @richard-cox ! Please be aware to the changes of `PodAffinity`, I'm not sure this is a good inspiration. But I have no idea that how can I change the labels when harvester using the `PodAffinity`. If you have any ideas, please let me know, thank you very much! cc @WuJun2016